docs/TrafficSplitting.md

```diff
@@ -0,0 +1,166 @@
+# Traffic Splitting
+
+kn can help you control which revisions get traffic in your Knative Service.
+
+A Knative Service has a traffic mapping, which is a mapping of revisions of the Service to percent amounts of traffic, along with the ability to create unique URLs for particular revisions. It also has the ability to assign traffic to the latest Revision, which can change as new revisions are created.
```

Comment:
Change “It also has ...” to “A Knative can have traffic be assign to the latest Revision...”